src/main/webapp/ratingScript.js

```diff
@@ -103,13 +102,32 @@ async function getLatestRating() {
     document.getElementById("difficulty").value = formInfo["difficulty"];
     document.getElementById("grade").value = formInfo["grade"];
     document.getElementById("translate").value = formInfo["translation"];
-
-    messageRetrievalElementContainer.classList.remove("alert-secondary");
-    messageRetrievalElementContainer.classList.add("alert-success");
     messageRetrievalElement.innerHTML = "Your form has been populated!";
   }
 }
 
+function validSubmission() {
+  const allFormFields = [
+    document.getElementById("term-input"),
+    document.getElementById("prof-input"),
+    document.getElementById("rating-term"),
+    document.getElementById("rating-prof"),
+    document.getElementById("hours"),
+    document.getElementById("difficulty"),
+    document.getElementById("grade"),
+  ];
+
+  );
+  for (let field of allFormFields) {
+    // If any field is not field out properly.
+    if (!field.checkValidity()) {
+      return false;
+    }
+    // If no field was submitted the wrong way.
+    return true;
```

Comment:
Shouldn't this be outside the for loop? Right now as soon as there's a valid field it's going to return true for the whole function.

Reply:
Ohh that's weird! I think this got modified by accident in my previous PR changing that now! :)